src/lib/components/file-browser/file-list/file-list.component.html

```diff
@@ -1,104 +1,191 @@
+        mat-icon-button
+        *ngIf="_parentFileBrowser.inputAllowFolders"
+        [disabled]="_selection.isEmpty()"
+        [tcTooltip]="_translationPrefix + '.deleteSelectedFiles' | translate: _locale.language"
+        (click)="_deleteSelected()"
+    >
+        <mat-icon>delete</mat-icon>
+    </button>
 
     <label style="width: 0; height: 0; visibility: hidden" #fileChooser>
         <input type="file" (change)="onFileSelect($event)" multiple="multiple" />
     </label>
+</mat-toolbar>
 
-    <!-- TODO is this needed???? -->
-    <!--<div class="loading-placeholder" *ngIf="!storageList">-->
-    <!--<p>{{ translationPrefix + '.loadingFiles' | translate:lang }}</p>-->
-    <!--<span class="icon-loading"></span>-->
-    <!--</div>-->
-</terra-base-toolbar>
-
-<div class="file-list-container" [class.empty]="_fileTableRowList.length <= 0">
-    <div [hidden]="!_showNewDirectoryPrompt" class="prompt" [class.in]="_showNewDirectoryPrompt">
-        <span class="prompt-title">{{ _translationPrefix + '.createFolder' | translate: _locale.language }}</span>
-        <terra-text-input
-            [inputName]="_translationPrefix + '.folderName' | translate: _locale.language"
-            [(ngModel)]="newDirectoryName"
-        >
-        </terra-text-input>
-        <div class="float-lg-right">
-            <terra-button
-                [inputIcon]="'icon-close'"
-                [inputCaption]="_translationPrefix + '.cancel' | translate: _locale.language"
-                [inputIsSmall]="true"
-                (outputClicked)="_showNewDirectoryPrompt = false"
-            ></terra-button>
+<div class="file-list-container">
+    <ng-template #newDirectoryPrompt>
+        <h2 mat-dialog-title>{{ _translationPrefix + '.createFolder' | translate: _locale.language }}</h2>
+        <mat-dialog-content>
+            <mat-form-field class="w-100">
+                <mat-label>{{ _translationPrefix + '.folderName' | translate: _locale.language }}</mat-label>
+                <input matInput [(ngModel)]="newDirectoryName" />
+            </mat-form-field>
+        </mat-dialog-content>
+        <mat-dialog-actions align="end">
+            <button mat-button [mat-dialog-close]="false">
+                {{ _translationPrefix + '.cancel' | translate: _locale.language }}
+            </button>
 
-            <terra-button
-                [inputIcon]="'icon-folder_add'"
-                [inputCaption]="_translationPrefix + '.create' | translate: _locale.language"
-                [inputIsSmall]="true"
-                [inputIsDisabled]="!newDirectoryName"
-                (outputClicked)="_createDirectory()"
-            ></terra-button>
-        </div>
-    </div>
+            <button mat-button [mat-dialog-close]="true" [disabled]="!newDirectoryName">
+                {{ _translationPrefix + '.create' | translate: _locale.language }}
+            </button>
+        </mat-dialog-actions>
+    </ng-template>
 
-    <terra-simple-table
+    <table
```

Comment:
hier fehlt noch um die Tabelle... `<div class="terra-table-inner-container">` (in Terra gibt es eine Directive dafür)